content/partials/examples/dimmer.slim

```diff
@@ -1,63 +1,64 @@
-.dimmer.demo
-  .ui.segment
-    h3.ui.header
-      |  Dimmable Section
-    .ui.small.images
-      img.ui.wireframe.image[src="/static/kitchen-sink/images/wireframe/image.png"]
-      img.ui.wireframe.image[src="/static/kitchen-sink/images/wireframe/image.png"]
-      img.ui.wireframe.image[src="/static/kitchen-sink/images/wireframe/image.png"]
-    img.ui.medium.wireframe.image[src="/static/kitchen-sink/images/wireframe/media-paragraph.png"]
-    .ui.dimmer
-  .ui.ignored.ignored.icon.buttons
-    .ui.show.button
+.ui
```

Comment:
Not sure why you add a single `.ui` element here. I've never seen a single `.ui` in semantic-ui. Since in semantic-ui, `.ui` is always combined with other class like `.button`, `.menu`, it's just a tag to show that the element is a UI element.